XiEditor/XiWindowController.swift

```diff
@@ -0,0 +1,10 @@
+import AppKit.NSWindowController
```

Comment:
You can copy over the header from one of the existing files (with the current year)